.github/workflows/test.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       - uses: erlef/setup-beam@v1.9.0
         with:
           otp-version: "23.2"
-          gleam-version: "0.20.0"
-      - run: gleam test
+          gleam-version: "0.21.0"
       - run: gleam format --check src test
+      - run: gleam deps download
+      - run: gleam test
```

Comment:
Can we run the tests for both the JavaScript and the Erlang targets here please

Reply:
Sure, I've added the command :)